addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1399,7 +1399,16 @@ function _createColumnGrid() {
  * @returns {Comment}
  */
 function _createMso(content='') {
-    return document.createComment(`[if mso]>${content}<![endif]`)
+    // We remove commets having opposite condition fron the one we will insert
+    // We remove comment tags having the same condition
+    const showRegex = /<!--\[if\s+mso\]>([\s\S]*?)<!\[endif\]-->/g;
+    const hideRegex = /<!--\[if\s+!mso\]>([\s\S]*?)<!\[endif\]-->/g;
+    let contentToInsert = content;
+    contentToInsert = contentToInsert.replace(showRegex, (matchedContent, p0) =>
+        matchedContent.slice(13, -12)
```

Comment:
You should used the group that was matched by `([\s\S]*?)` instead of the whole match so that you wouldn't need to slice manually.

Reply:
That's smart, thanks !